### PR TITLE
Remove tooltip from body on tooltip.destroy() if it was appended to body by the tooltip

### PR DESCRIPTION
**Pull request by maxcbc**
Remove tooltip from body on tooltip.destroy() if it was appended to body by the tooltip

This is just making the Tooltip clean up after itself

### CODE REVIEW

File: src/js/tooltip.js [hunk 1]
```diff
@@ -244,6 +244,10 @@ class Tooltip {
 			this.close();
 		}
 
+		if (this.opts.appendToBody) {
```
Comment (notlee) [inline src/js/tooltip.js]: Thanks for the PR, the tooltip element should definitely be removed on `destroy`. I think it should be removed always, not just when `this.opts.appendToBody` is chosen?

Comment (maxcbc) [reply src/js/tooltip.js]: My instinct is that it should only be removed if the tooltip itself created it, if it was already in the dom it should be left.

Comment (maxcbc) [reply src/js/tooltip.js]:
```suggestion
		if (this.opts && this.opts.content) {
```

Comment (maxcbc) [reply src/js/tooltip.js]: Maybe the above??

**Comment by notlee** [comment]
Looks good to me! Would you mind adding a test in`Tooltip.test.js`?

**Comment by maxcbc** [comment]
@notlee Have added a test

**Pull request merged**